docs/2-linguistics/2-canopy-ai.md

```diff
@@ -49,18 +49,18 @@ In OpenShift, you have an experimentation environment which is called `<USER_NAM
 
 4. Hit `Create` and fill out the values as below:
 
-    - SYSTEM_PROMPT: Do you remember the great prompt you came up with in the previous section. Let's paste that here!
-    - MODEL_NAME: `llama32`
-    - LLM_ENDPOINT: `https://llama32-ai501.<CLUSTER_DOMAIN>`
+    - **SYSTEM_PROMPT:** Do you remember the great prompt you came up with in the previous section? Let's paste that here!
+    - **MODEL_NAME:** `llama32`
+    - **LLM_ENDPOINT:** `https://llama32-ai501.<CLUSTER_DOMAIN>`
   
     Leave the rest as it is for now.
 
     ![helm-values.png](./images/helm-values.png)
 
     ✅ This will create:
 
-    - A UBI9-based Streamlit application that sends your chat requests to the LLM
-    - A secure OpenShift route for us to access the app outside of the cluster
+      - A UBI9-based Streamlit application that sends your chat requests to the LLM
+      - A secure OpenShift route for us to access the app outside of the cluster
 
 5. Once the application is successfully running, click on the arrow on the side of the circle to access the Canopy UI 🌳🌳🌳
```

Comment:
when we're working with the Summarization UI, it's not referring to the tea in the image. Shall we use the text for the Canopy or reuse the tea one? @RHRolun @ckavili